src/patternfly/components/Timestamp/timestamp.scss

```diff
@@ -0,0 +1,43 @@
+.pf-c-timestamp {
+  // Timestamp text
+  --pf-c-timestamp__text--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-timestamp__text--Color: var(--pf-global--Color--200);
```

Comment:
I think you can set these props on `pf-c-timestamp` as they're inheritable and any other elements/icons within timestamp would likely need these values as well. Otherwise you would have a dark icon next to lighter text. @mcoker wdyt?